README.md

```diff
@@ -154,38 +229,66 @@ No modules.
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
+| <a name="input_access_key_max_age"></a> [access\_key\_max\_age](#input\_access\_key\_max\_age) | Maximum number of days without rotation. | `number` | `90` | no |
```

Comment:
In rich preview mode, quite a few of these look like they're not on their own row, rather a few are on all the same row. Not sure where or how the formatting got off.

Reply:
we might have to get together on this one as I am not seeing what you are seeing.

Reply:
Looks OK to me. A screenshot might help?

Reply:
Now I'm getting an error in the UI trying to view the preview, but if I click `View File` I get [this](https://github.com/trussworks/terraform-aws-config/blob/90e6e9686580571d04a015c0bfc3135da62f5695/README.md) which looks perfect!